crates/extensions/tedge_log_manager/src/actor.rs

```diff
@@ -1,45 +1,51 @@
+use std::collections::HashMap;
+
 use async_trait::async_trait;
-use http::status::StatusCode;
+use camino::Utf8Path;
 use log::debug;
 use log::error;
 use log::info;
+use log::warn;
 use log_manager::LogPluginConfig;
 use serde_json::json;
 use tedge_actors::fan_in_message_type;
 use tedge_actors::Actor;
 use tedge_actors::ChannelError;
-use tedge_actors::ClientMessageBox;
+use tedge_actors::DynSender;
 use tedge_actors::LoggingSender;
 use tedge_actors::MessageReceiver;
 use tedge_actors::NoMessage;
 use tedge_actors::RuntimeError;
 use tedge_actors::Sender;
 use tedge_actors::SimpleMessageBox;
+use tedge_api::messages::CommandStatus;
+use tedge_api::messages::LogUploadCmdPayload;
 use tedge_api::Jsonify;
 use tedge_file_system_ext::FsWatchEvent;
-use tedge_http_ext::HttpError;
-use tedge_http_ext::HttpRequest;
-use tedge_http_ext::HttpRequestBuilder;
-use tedge_http_ext::HttpResponseExt;
-use tedge_http_ext::HttpResult;
 use tedge_mqtt_ext::MqttMessage;
 use tedge_mqtt_ext::Topic;
+use tedge_uploader_ext::UploadRequest;
+use tedge_uploader_ext::UploadResult;
 
 use super::error::LogManagementError;
 use super::LogManagerConfig;
 use super::DEFAULT_PLUGIN_CONFIG_FILE_NAME;
-use tedge_api::messages::CommandStatus;
-use tedge_api::messages::LogUploadCmdPayload;
 
-fan_in_message_type!(LogInput[MqttMessage, FsWatchEvent] : Debug);
-fan_in_message_type!(LogOutput[MqttMessage]: Debug);
+type MqttTopic = String;
```

Comment:
suggestion: Not all strings are valid MQTT topics, so I'd rather see a type like `Topic` used here if possible.